feconf.py

```diff
@@ -257,6 +257,10 @@
 # schema changes.
 CURRENT_PLATFORM_PARAMETER_RULE_SCHEMA_VERSION = 1
 
+# This value is used to initialize the version number for the versioned
+# entities.
+INIT_VERSION = 0
```

Comment:
init_version sounds more like a function name. Can we call this something like default_version maybe?

Reply:
`default_version` doesn't explains that this number is used for initializing the version number. should I call it `INITIAL_VERSION`?

Reply:
Initial version sounds good--better to avoid the slight ambiguity of 'init'.

Reply:
Done!